src/main/java/server/db/UserDB.java

```diff
@@ -0,0 +1,167 @@
+package server.db;
+
+import server.models.User;
+import org.mindrot.jbcrypt.BCrypt;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+
+public class UserDB {
+    private static Connection mConn;
+
+    static {
+        try {
+            Class.forName("org.postgresql.Driver");
+            String url = "jdbc:postgresql://localhost:5432/spotifylite";

```

Comment:
This would probably be best defined as a shared constant somewhere. It's also used in other DB classes.